client/vscode/package.json

```diff
@@ -40,6 +40,10 @@
             {
                 "command": "helloWorld.log",
                 "title": "Hello World LSP - Log Command"
+            },
+            {
+                "command": "aws/qNetTransform/startTransform",
```

Comment:
nit: can it follow the same pattern as above, with dots?

Reply:
sure